### PR TITLE
fix typo and update comments

### PR DESCRIPTION
**Pull request by hezhizhen**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

1. fix some typos
2. update comments to meet Go's needs

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hezhizhen** [comment]
There are 2 commits: the first one does what's described in the title and the second one is the result of `make generate` and `make verify`.

**Review by wackxu — COMMENTED** [review]
LGTM

File: build/crds/apps/apps_v1alpha1_edgeapplication.yaml [hunk 1]
```diff
@@ -1,10 +1,9 @@
-
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```
Comment (fisherxu) [inline build/crds/apps/apps_v1alpha1_edgeapplication.yaml:6]: Is it possible to upgrade the kubebuilder version in another PR? In this PR, let's focus on fixing the typo and comments.

**Review by fisherxu — COMMENTED** [review]
Thanks for the fixing :) @hezhizhen

File: build/crds/apps/apps_v1alpha1_edgeapplication.yaml (hunk 1 above)
Comment (hezhizhen) [reply build/crds/apps/apps_v1alpha1_edgeapplication.yaml:6]: okay.

Comment (hezhizhen) [reply build/crds/apps/apps_v1alpha1_edgeapplication.yaml:6]: But I need to revert the second commit, which ran `make generate verify`. Will the CI pass without this commit?

Comment (hezhizhen) [reply build/crds/apps/apps_v1alpha1_edgeapplication.yaml]: Done.

**Comment by hezhizhen** [comment]
@fisherxu PTAL

**Comment by hezhizhen** [comment]
@fisherxu PTAL

**Comment by hezhizhen** [comment]
@fisherxu @wackxu

**Review by fisherxu — COMMENTED** [review]
@hezhizhen Thanks for the fixing! Overall LGTM.

And because you modified the api comments, so it also need to update the crd defination yaml.

The ci err msg: `hack/../build/crds is out of date. Please run hack/generate-crds.sh`

So please run `hack/generate-crds.sh` in your host, and submit the modified yamls :)

**Comment by hezhizhen** [comment]
> @hezhizhen Thanks for the fixing! Overall LGTM.
> 
> 
> 
> And because you modified the api comments, so it also need to update the crd defination yaml.
> 
> 
> 
> The ci err msg: `hack/../build/crds is out of date. Please run hack/generate-crds.sh`
> 
> 
> 
> So please run `hack/generate-crds.sh` in your host, and submit the modified yamls :)

Thanks for your advice. I'll run it later.

**Comment by hezhizhen** [comment]
Done @fisherxu

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**